src/scene_battle.cpp

```diff
@@ -89,7 +90,9 @@ void Scene_Battle::Start() {
 		return;
 	}
 
-	Output::Debug("Starting battle {} ({})", troop_id, troop->name);
+	autobattle_algo = AutoBattle::CreateAlgorithm(Player::player_config.autobattle_algo.Get());
+
+	Output::Debug("Starting battle {} ({}) : autobattle={}", troop_id, troop->name, autobattle_algo->GetName());
```

Comment:
isn't it a bit verbose to print the constant autobattle algo everytime a battle starts?

btw is the algorithm name case sensitive or can I also specify "attack" on the command line?

Reply:
My original design was printing this once on startup in it's own logfile line. I was thinking of future extensibility where you could change the algo at runtime, but since we don't have that yet this is a bit overkill.

The algo name is case sensitive, I can fix that

Reply:
Added the case insensitive name compare. I left the log message in as it seems to me it'll be better for debugging when the algo is changeable in scene settings etc..

That being said if you feel strongly I'll remove it.

Reply:
well keep it but please get rid of the whitespace before the ":". (Also imo a comma or a semi-colon would fit better here.)